src/lib/keyseq.ts

```diff
@@ -484,25 +478,111 @@ function numericPrefixToExstrSuffix(numericPrefix: MinimalKey[]) {
     }
 }
 
+const keyWithShiftMap = {
+    a: "A",
+    b: "B",
+    c: "C",
+    d: "D",
+    e: "E",
+    f: "F",
+    g: "G",
+    h: "H",
+    i: "I",
+    j: "J",
+    k: "K",
+    l: "L",
+    m: "M",
+    n: "N",
+    o: "O",
+    p: "P",
+    q: "Q",
+    r: "R",
+    s: "S",
+    t: "T",
+    u: "U",
+    v: "V",
+    w: "W",
+    x: "X",
+    y: "Y",
+    z: "Z",
+    "`": "~",
+    "1": "!",
+    "2": "@",
+    "3": "#",
+    "4": "$",
+    "5": "%",
+    "6": "^",
+    "7": "&",
+    "8": "*",
+    "9": "(",
+    "0": ")",
+    "-": "_",
+    "=": "+",
+    "[": "{",
+    "]": "}",
+    "\\": "|",
+    ";": ":",
+    "'": '"',
+    ",": "<",
+    ".": ">",
+    "/": "?",
+}
+
+const keycodetranslatemap = {
```

Comment:
We should link to the page they came from (I imagine https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_code_values#code_values_on_linux_x11 ?) and triple check to make sure all pressable keys are present.

We're missing `Quote` for example which lots of layouts change :)

Reply:
Sure, i missed `Quote`, brackets and minus. Link may be useful too, even though i generated this map just by doing  
``console.log(`{e.code}: "{e.key}",`)``on en layout and a bit of python for letters and digits.